SwiftCompilerSources/Sources/Optimizer/Utilities/OptUtils.swift

```diff
@@ -396,3 +396,21 @@ extension Function {
   }
 }
 
+extension GlobalVariable {
+  func stripAccessInstructionFromInitializer(_ context: FunctionPassContext) {
```

Comment:
Could an explanation for why it's legal to delete all the begin_access/end_access instructions be helpful?